protocol.html

```diff
@@ -572,7 +572,7 @@ <h4 property="schema:name">Description Resource</h4>
 
                       <p>Servers MUST NOT directly associate more than one description resource to a subject resource.</p>
 
-                      <p>When a HTTP request targets a description resource, the server MUST apply the authorization policy that is used for the subject resource that the description resource is associated with.</p>
+                      <p>When an HTTP request targets a description resource, the server MUST apply the authorization policy that is used for the subject resource with which the description resource is associated.</p>
```

Comment:
Could we simplify further?
"A subject resource's authorization MUST apply to its description resource."

Reply:
That implies that the resource's authorization is the thing taking action.

The imperative is on the server, to apply the same authorization as is on the resource to the description resource(s), and I think we've got just about the simplest, *clearest* possible wording.